README.md

```diff
@@ -1,2 +1,33 @@
 # helm-charts
-Helm charts for deploying SDF maintained software
+
+This repository contains the following Helm charts for deploying SDF maintained software:
+- Anchor Platform SEP Service
+
+# Anchor Platform SEP Service
+Documentation
+https://github.com/stellar/java-stellar-anchor-sdk/tree/main/docs
+
+## Add Repository (stable)
+```
+$ helm repo add sdf-stable https://helm.sdf.org/stable
```

Comment:
What's the plan with regards to the stable/edge split? Do you plan to use "edge" for development and move to stable when things are ready for golive?

Also the domain we use is `*.stellar.org` rather than `*.sdf.org`. So maybe the repo could be `https://helm.stellar.org/stable`?

Reply:
yea, right now the charts in edge are experimental and not production.  we are currently testing chart 0.3.xx with our design partners for anchor platform.  this pattern i saw with nginx where they release production. The hosted repo could also contain the chart packages in edge (expermental) and stable.  I saw this pattern for nginx charts.

i removed the domain stuff for now. I guess we should add that once its setup with instructions on how to install from the chat.

I have updated my PR with the helm chart source for SEP service.  I have tested the instructions in sandbox and ran validation test

ip-192-168-1-106~/dev/helm-charts % helm install -f values.yaml anchorplatform ./edge/anchor-platform/sep-service --namespace sandbox

WARNING: Kubernetes configuration file is group-readable. This is insecure. Location: /Users/reecemarkowsky/.kube/config
WARNING: Kubernetes configuration file is world-readable. This is insecure. Location: /Users/reecemarkowsky/.kube/config
NAME: anchorplatform
LAST DEPLOYED: Wed Jul 20 20:44:15 2022
NAMESPACE: sandbox
STATUS: deployed
REVISION: 1
TEST SUITE: None


NAME                     READY   STATUS    RESTARTS   AGE
reecekafka-0             1/1     Running   1          2d3h
reecekafka-zookeeper-0   1/1     Running   0          2d3h
sep-65f584d97b-rrmmx     1/1     Running   0          2m13s


Reeces-MacBook-Pro~/sdf-helm %  yarn stellar-anchor-tests -v --seps 1 10 12 31 -c ~/config/config2.json --home-domain https://reece.sandbox.kube001.services.stellar-ops.com
yarn run v1.22.19
warning package.json: No license field
$ /Users/reecemarkowsky/node_modules/.bin/stellar-anchor-tests -v --seps 1 10 12 31 -c /Users/reecemarkowsky/config/config2.json --home-domain https://reece.sandbox.kube001.services.stellar-ops.com
✔ SEP-1 › TOML Tests › the TOML file exists at ./well-known/stellar.toml
✔ SEP-1 › TOML Tests › the file has a size less than 100KB
✔ SEP-1 › TOML Tests › has a valid network passphrase
✔ SEP-1 › TOML Tests › has a valid CURRENCIES section
✔ SEP-1 › TOML Tests › all URLs are HTTPS and end without slashes
✔ SEP-10 › TOML Tests › has a valid WEB_AUTH_ENDPOINT in the TOML file
✔ SEP-10 › TOML Tests › has valid SIGNING_KEY
✔ SEP-10 › GET /auth › returns a valid GET /auth response
✔ SEP-10 › GET /auth › rejects requests with no 'account' parameter
✔ SEP-10 › GET /auth › rejects requests with an invalid 'account' parameter
✔ SEP-10 › POST /auth › returns a valid JWT
✔ SEP-10 › POST /auth › accepts JSON requests
✔ SEP-10 › POST /auth › fails with no 'transaction' key in the body
✔ SEP-10 › POST /auth › fails if the challenge is not signed by the client
✔ SEP-10 › POST /auth › fails if the 'transaction' value is invalid
✔ SEP-10 › POST /auth › fails if the challenge is not signed by SIGNING_KEY
✔ SEP-10 › POST /auth › fails if a challenge for a nonexistent account has extra client signatures
✔ SEP-10 › Account Signer Support › fails if the challenge signature weight is less than the account's medium threshold
✔ SEP-10 › Account Signer Support › succeeds with a signature from a non-master signer
✔ SEP-10 › Account Signer Support › fails for challenges signed more than once by the same signer
✔ SEP-10 › Account Signer Support › returns a token for challenges with sufficient signatures from multiple non-master signers
✔ SEP-12 › TOML Tests › has KYC_SERVER attribute
✔ SEP-12 › PUT /customer › requires a SEP-10 JWT
✔ SEP-12 › PUT /customer › can create a customer
✔ SEP-12 › PUT /customer › memos differentiate customers registered by the same account
✔ SEP-12 › GET /customer › requires a SEP-10 JWT
✔ SEP-12 › GET /customer › has a valid schema for a new customer
✔ SEP-12 › GET /customer › can retrieve customer using 'id'
✔ SEP-12 › GET /customer › can retrieve customer using SEP-10 token
✔ SEP-12 › DELETE /customer › requires a SEP-10 JWT
✔ SEP-12 › DELETE /customer › can delete a customer
✔ SEP-31 › TOML Tests › has DIRECT_PAYMENT_SERVER attribute
✔ SEP-31 › GET /info › matches the expected schema
✔ SEP-31 › GET /info › has expected asset enabled
✔ SEP-31 › GET /info › has configured transaction 'fields'
✔ SEP-31 › POST /transactions › requires a SEP-10 JWT
✔ SEP-31 › POST /transactions › can create a transaction
✔ SEP-31 › POST /transactions › returns 400 when no amount is given
✔ SEP-31 › POST /transactions › returns 400 when no asset_code is given
✔ SEP-31 › GET /transactions/:id › can fetch a created transaction
✔ SEP-31 › GET /transactions/:id › response body complies with protocol schema
✔ SEP-31 › GET /transactions/:id › returns 404 for a non-existent transaction

Tests:       42 passed, 42 total
Time:        63.173s
✨  Done in 63.73s.
Reeces-MacBook-Pro~/sdf-helm %

Reply:
> helm install -f values.yaml anchorplatform ./edge/anchor-platform/sep-service --namespace sandbox

i removed the edge/stable as this made more sense when it was hosting chart packages - doesn't really lend it self to chart source.